exp/operator/controllers/suite_test.go

```diff
@@ -43,9 +49,20 @@ func TestAPIs(t *testing.T) {
 }
 
 func TestMain(m *testing.M) {
+	os.Setenv("TEST_ASSET_KUBECTL", "/tmp/kubebuilder/bin/kubectl")
+	os.Setenv("TEST_ASSET_KUBE_APISERVER", "/tmp/kubebuilder/bin/kube-apiserver")
+	os.Setenv("TEST_ASSET_ETCD", "/tmp/kubebuilder/bin/etcd")
```

Comment:
I there a particular reason why we are enforcing this (we are not doing this in other tests...)

Reply:
no, some leftovers after my local debugging. Will remove it.